docs/src/assets/scss/syntax-highlighter.scss

```diff
@@ -106,6 +106,12 @@ pre[class*="language-"] {
     cursor: help;
 }
 
+.language-abnf .token {
+    display: inline;
+    padding: 0;
+    margin: 0;
+}
+
```

Comment:
To fix formatting on the web site: https://eslint.org/docs/latest/developer-guide/package-json-conventions